modules/threescale-backend-cache.adoc

```diff
@@ -0,0 +1,44 @@
+
+== Advantages of enabling backend cache
+
+The following are advantages to enabling the backend cache:
+
+* Enable the backend cache when you find latencies are high while accessing services managed by the 3scale Istio Adapter.
+* Enabling the backend cache will stop the adapter from continually checking with the 3scale API manager for request authorizations, which will lower the latency.
+** This creates an in-memory cache of 3scale authorizations for the 3scale Istio Adapter to store and reuse before attempting to contact the 3scale API manager for authorizations. Authorizations will then take much less time to be granted or denied.
+* Backend caching is useful in cases when you are hosting the 3scale API manager in another geographical location from the service mesh running the 3scale Istio Adapter.
+** This is generally the case with the 3scale Hosted (SaaS) platform, but also if a user hosts their 3scale API manager in another cluster located in a different geographical location, in a different availability zone, or in any case where the network overhead to reach the 3scale API manager is noticeable.
+
+
+== Trade-offs for having lower latencies
+
+The following are trade-offs for having lower latencies:
+
+* Each 3scale adapter's authorization state updates every time a flush happens.
+** This means two or more instances of the adapter will introduce more inaccuracy between flushing periods.
+** There is a greater chance of too many requests being granted that exceed limits and introduce erratic behavior, which leads to some requests going through and some not, depending on which adapter processes each request.
+* An adapter cache that cannot flush its data and update its authorization information risks shut down or crashing without reporting its information to the API manager.
+* A fail open or fail closed policy will be applied when an adapter cache cannot determine whether a request must be granted or denied, possibly due to network connectivity issues in contacting the API manager.
+* When cache misses occur, typically right after booting the adapter or after a long period of no connectivity, latencies will grow in order to query the API manager.
+* An adapter cache must do much more work on computing authorizations than it would without an enabled cache, which will tax processor resources.
+* Memory requirements will grow proportionally to the combination of the amount of limits, applications, and services managed by the cache.
+
+== Backend cache configuration settings
+
+The following points explain the backend cache configuration settings:
+
+* Find the settings to configure the backend cache in the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.6/html-single/service_mesh/index#ossm-cr-threescale_customizing-installation-ossm[Example 3scale configuration options].
+* The last 3 settings control enabling of backend cache:
+** Use cache backend - set to `true` to enable backend cache.
```

Comment:
Should these match the actual parameters to set? 

i.e.
`USE_CACHE_BACKEND` - set to `true` to enable backend cache.